app/docs/how-to/streamline-curbside.md

```diff
@@ -0,0 +1,255 @@
+	<img src="/images/how-tos/curbside-pickup/location-drag-drop.png" width="90%">
+</p>
+
+Then, give your new location a name, description (optional) and click **Create Location** button.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/location-dialog.png" width="75%">
+</p>
+
+#### Step 3: Create a new touchpoint
+
+Create a new touchpoint by dragging a touchpoint icon into a desired location. In this example, we are putting it under the *Parking* location created in [Step 2](#step-2-create-a-new-location).
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/touchpoint-drag-drop.png" width="90%">
+</p>
+
+Then, define name and description for your new touchpoint.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/touchpoint-name-description.png" width="75%">
+</p>
+
+Routegy will use the name and description in the header of your new touchpoint form as its title and subtitle respectively (see screenshot below). With this in mind, try using short, concise language to provide clear instructions to your customers (e.g. a simple call to to action).
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/curbside-form-name-description.png" width="30%">
+</p>
+
+#### Step 4: Define touchpoint schema
+
+Next, we are going to define a touchpoint [schema](/reference/schemas/). Schema is an exteremely important part of a touchpoint that defines what kind of data is collected from people interacting with your touchpoint.
+
+To get started, select **Define your own schema** from **Schema** dropdown list and name your new schema.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/touchpoint-define-schema.png" width="75%">
+</p>
+
+The Routegy schema is defined by a [JSON Schema](https://json-schema.org/) document in YAML format. To collect customer's name, car information and order type, we are going to use the following schema:
+
+```yaml
+type: object
+attrs:
+  successMessage: Thank You!
+order:
+  - order_type
+  - name
+  - car
+required:
+  - name
+  - car
+  - order_type
+properties:
+  car:
+    type: string
+    attrs:
+      placeholder: E.g. White Honda Accord
+    title: Please give us some details about your car
+  name:
+    type: string
+    attrs:
+      placeholder: E.g. John
+    title: What is the name on the order?
+  order_type:
+    enum:
+      - By Phone
+      - Online
+      - GrubHub
+      - Other
+    type: string
+    attrs:
+      type: radio
+    title: How did you place your order?
+```
+
+This YAML document defines a simple schema with **car**, **name** and **order_type** fields, and it can be easily tailored to meet the needs of your business. For instance, you can easily modify titles of these fields by changing **title** attribute under each individual field.
+
+To apply this document to our newely created schema, scroll down to **Definition (YAML)** section of the form, and copy/paste the YAML code above replacing the original content.
```

Comment:
1) What does "above replacing the original content" mean?

2) "newely" -> "newly"

3) Cut down version:

> To apply this document to our newly created schema, copy/paste the YAML into the **Definition (YAML)** section of the form.

Reply:
Re #1: wanted to be explicit about replacing the default, example schema YAML, not appending to it, or pasting it randomly in the middle of it.

Reply:
Add made into this: 

> To apply this document to our newly created schema, scroll down to **Definition (YAML)** section of the form, and copy/paste the YAML code above replacing the default schema YAML.